docs/general/configuration_options.md

```diff
@@ -5,6 +5,7 @@ The following configuration options can be set on `ActiveModel::Serializer.confi
 ## General
 
 - `adapter`: The [adapter](adapters.md) to use. Possible values: `:attributes, :json, :json_api`. Default: `:attributes`.
+- `automatic_lookup`: Whether serializer should be automatically looked up or manually provided. Default: `true`
```

Comment:
If that's not sufficient, that's the area of the public API that should be affected.. would love to see a code example of what you're trying to do